app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsFragment.kt

```diff
@@ -436,6 +440,7 @@ class ChatRoomsFragment : Fragment(), ChatRoomsView {
 
     override fun hideLoading() {
         view_loading.isVisible = false
+        clickDisabled = false
```

Comment:
why do you put the resetting of this flag in hideLoading? It seems a little non-inuitive, though it does work in my testing.

Reply:
Since ChatRoomsPresenter [here](https://github.com/WideChat/Rocket.Chat.Android/blob/cc3521ed902aca533a8d7cf33f5296a3888462f6/app/src/main/java/chat/rocket/android/chatrooms/presentation/ChatRoomsView.kt) is inherited from LoadingView, it calls hideLoading callback when it finished opening a room. So, in that callback we'll want the click to be enabled again.

Reply:
is it guaranteed to be called?

Reply:
It should be, as loading is hided when room opens or fails to open.